scripts/prerelease.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+echo "Select an option for release："
+echo
+
+select VERSION in patch minor major "Specific Version"
+  do
+    echo
+    if [[ $REPLY =~ ^[1-4]$ ]]; then
+      if [[ $REPLY == 4 ]]; then
+        read -p "Enter a specific version: " -r VERSION
+        echo
+        if [[ -z $REPLY ]]; then
+          VERSION=$REPLY
+        fi
+      fi
+
+      read -p "Release $VERSION - are you sure? (y/n) " -n 1 -r
+      echo
+
+      if [[ $REPLY =~ ^[Yy]$ || -z $REPLY ]]; then
+        # bump version
+        yarn version --new-version $VERSION --no-git-tag-version
+        NEW_VERSION=$(node -p "require('./package.json').version")
+
+        # create new branch
+        git checkout -b $NEW_VERSION master
+
+        # cut docusaurus docs version
+        cd website && yarn run version $NEW_VERSION
```

Comment:
Omg we should probably rename this command within `website/package.json`. It's easy to confuse with yarn's native `version` command.

Reply:
Will `docsversion` do?

Reply:
Sounds good. Will need to update a few places as well.

Reply:
Actually i wanted to rework all the CLI commands to `docusaurus <command>` instead of `docusaurus-command` but this might lead to breaking changes.

Reply:
Oh yes we should do that, but not now. I fully agree with you (wrote about that in [v2 under API](https://github.com/facebook/Docusaurus/issues/789)) also.